src/events.js

```diff
@@ -3866,15 +3861,15 @@ const addDeviationTypeCommentsContent = (searchSpecimenInstituteArray, currTube,
 
         if (acceptedDeviationArray.length >= 1) {
             for (const deviationLabel of acceptedDeviationArray) {
-                deviationString += `${deviationLabel} <br><br>`;
+                deviationString += `${deviationLabel} <br>`;
             }
             deviationTypeCell.innerHTML = deviationString;
         } else {
-            deviationTypeCell.innerHTML = `<br><br>`;
+            deviationTypeCell.innerHTML = `<br>`;
         }
         commentCell.innerHTML = currTubeComments;
     }
-    if (bagsArrayIndex % 2 == 0) {
+    if (bagsArrayIndex % 2 === 0) {
```

Comment:
Related: this effect (different backgrounds in odd and even rows) can be achieved using CSS (nth-child)

Reply:
This is a good suggestion, but I don't think I can apply `nth-child`. The first `<tr>` child are the column names. The remaining `<tr>` backaground color is dependent on the array index in the `bagsArrayIndex` value. Maybe I can assign a class to currRow, `currRow.classList.add("manifestRowBackgroundColor")`, then provide css styling in the css file

**First <tr> child of second image (tr with table headers)**
<img width="500" alt="Screenshot 2023-06-09 at 4 52 55 PM" src="https://github.com/episphere/biospecimen/assets/33293205/f1351622-25ab-4d89-ba1c-825555544c52">

**Child elements of table body**
<img width="500" alt="Screenshot 2023-06-09 at 4 22 09 PM" src="https://github.com/episphere/biospecimen/assets/33293205/47aa8960-7ffb-4d2c-a63b-05d79b5d9e71">

Reply:
No worries. Current solution works well and there's no need to change. I put the info just for technical discussion. We can check together later if time permitted.